socorro/processor/mozilla_transform_rules.py

```diff
@@ -151,6 +151,16 @@ class AddonsRule(Rule):
     def version(self):
         return '1.0'
 
+    def _get_formatted_addon(self, addon):
+        """Return a properly formatted addon string.
+
+        Format is: addon_identifier:addon_version
+
+        This is used because some addons are missing a version. In order to
+        simplify subsequent queries, we make sure the format is consistent.
+        """
+        return ':' in addon and addon or addon + ':NO_VERSION'
```

Comment:
Python has a "ternary" format. This would be better written as:

```
return addon if ':' in addon else ':NO_VERSION'
```

It reads a little weird if you're used to ternary operators in other languages, but the semantics are more correct and handle non-string values better.